fv3gfs-physics/fv3gfs/physics/global_constants.py

```diff
@@ -129,7 +129,7 @@
 mono_prof = False  # Perform terminal fall with mono ppm scheme
 mp_time = 225.0  # Maximum microphysics timestep (sec)
 prog_ccn = 0  # Do prognostic ccn (yi ming's method)
-qi0_crt = 8e-05  # Cloud ice to snow autoconversion threshold (highly dependent on horizontal resolution)
+qi0_crt = 8e-05  # Cloud ice to snow autoconversion threshold
```

Comment:
is it OK to remove that from this comment? You could continue the comment on the next line.